gcalendar-new-event-to-trello-card/main.bal

```diff
@@ -0,0 +1,57 @@
+import ballerina/http;
+import ballerinax/trello;
+import ballerinax/trigger.google.calendar;
+
+listener http:Listener httpListener = new (8090);
+
+// Google Calendar configuration parameters
+configurable calendar:ListenerConfig config = ?;
+
+// Trello configuration parameters
+configurable string trelloApiKey = ?;
+configurable string trelloApiToken = ?;
+configurable string trelloListId = ?;
+
+listener calendar:Listener calendarListener = new (config, httpListener);
```

Comment:
Shall we move the Calendar listener related configs above near to the service.

Reply:
Fixed with 686f1420fcca26c7b3fc489f6cb12737010c5a0f